python/lsst/analysis/tools/actions/plot/interpolateDetectorPlot.py

```diff
@@ -51,34 +52,61 @@ class InterpolateDetectorMetricPlot(PlotAction):
 
     def getInputSchema(self) -> KeyedDataSchema:
         base = []
-
         base.append(("x", Vector))
         base.append(("y", Vector))
-        base.append(("metricValues", Vector))
+        for metricName in self.metricNames:
+            base.sappend((metricName, Vector))
 
         return base
 
     def __call__(self, data: KeyedData, **kwargs) -> Mapping[str, Figure] | Figure:
         return self.makePlot(data, **kwargs)
 
     def makePlot(self, data: KeyedData, plotInfo: Optional[Mapping[str, str]] = None, **kwargs) -> Figure:
```

Comment:
Please can you add full and detailed doc strings? The doc pages are generated from the docs in the code and are really helpful for people looking for the right plots to use. Preferably the docs would also include an example plot.

Reply:
This still needs a docs section, maybe have a look at scatterPlotWithTwoHists and follow that example.